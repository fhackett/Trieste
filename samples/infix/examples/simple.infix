x = 5 = 5;
print "x" x;
y = 2 - 1;
print "1 + 10" 1 + 10;
