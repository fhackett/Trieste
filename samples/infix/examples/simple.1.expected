//! calculate
//! --enable-tuples calculate
//! --enable-tuples --tuples-require-parens calculate
//! --enable-tuples --tuples-require-parens --use-parser-tuples calculate
(top
  {}
  (infix-calculation
    {}
    (infix-output
      (infix-string 1:x)
      (infix-literal
        (infix-int 1:5)))
    (infix-output
      (infix-string 6:1 + 10)
      (infix-literal
        (infix-int 2:11)))))
